personal/DariuszRatajczak/Reports/Zajecia3.Rmd

```diff
@@ -0,0 +1,87 @@
+---
+title: "Zajecia3"
+author: "Dariusz Ratajczak"
+date: "26 04 2020"
+output: html_document
+---
+
+```{r setup, include = FALSE}
+library(sf)
+library(fs)
+library(tidyverse)
+library(ggplot2)
+library(raster)
+library(devtools)
+library(spData)
+library(dplyr)
+library(lwgeom)
+knitr::opts_chunk$set(echo = FALSE, message = FALSE, warning = FALSE)
+```
+
+## ZADANIA

```

Comment:
Jak chcesz, to możesz ZADANIA zrobić pod jednym #: 
# ZADANIA, 
im więcej masz ### tym niżej schodzą w nagłowkach, to tak jak w html-u jest h1, h2, h3 ;)